afs/models.py

```diff
@@ -200,7 +200,7 @@ def upload_model(
             if "data" in pai_data_dir:
                 data = pai_data_dir["data"]
                 if "machineIdList" in data:
-                    machineIdList = data["machineIdList"].pop(0, None)
+                    machineIdList = data["machineIdList"].pop(0) if len(data["machineIdList"]) else None
```

Comment:
In that case, I would merge #202 to make it more simplified:
```python
machineIdList = data.get("machineIdList", [None]).pop(0)
```

Reply:
Fix it.